flows-starter/rest-api/interact-with-rest-api.py

```diff
@@ -0,0 +1,17 @@
+"""
+Discourse:
+https://discourse.prefect.io/t/how-can-i-interact-with-the-backend-api-using-a-python-client/80
+"""
+
```

Comment:
We could mention that its possible to do hit the api from a synchronous context with `sync_compatible`:
```python
from prefect.utilities.asyncutils import sync_compatible

@sync_compatible
async def trigger_flow_run(deployment_id: str, params: dict) -> FlowRun:
    async with get_client() as client:
        # do things with the client, like for example
        return await client.create_flow_run_from_deployment(
            deployment_id=deployment_id, parameters=params
        )
        
trigger_flow_run(
    deployment_id="XXXXXX",
    params={'arg': 'value'}
)
```